contribute/design/1.x/apis/cstorvolume_policy.md

```diff
@@ -117,49 +110,42 @@ type CStorVolumeConfigStatus struct {
 
 	// PoolInfo represents current pool names where volume replicas exists
 	PoolInfo []string `json:"poolInfo"`
-	
-  // Capacity the actual resources of the underlying volume.
-	Capacity   corev1.ResourceList         `json:"capacity,omitempty"`
-	
-  Conditions []CStorVolumeConfigCondition `json:"condition,omitempty"`
+
+	// Capacity the actual resources of the underlying volume.
+	Capacity corev1.ResourceList `json:"capacity,omitempty"`
+
+	Conditions []CStorVolumeConfigCondition `json:"condition,omitempty"`
 }
 
 // CStorVolumeConfigCondition contains details about state of cstor volume
 type CStorVolumeConfigCondition struct {
 	// Current Condition of cstor volume claim. If underlying persistent volume is being
 	// resized then the Condition will be set to 'ResizeStarted' etc
 	Type CStorVolumeConfigConditionType `json:"type"`
-	
-  // Last time we probed the condition.
+	// Last time we probed the condition.
 	// +optional
 	LastProbeTime metav1.Time `json:"lastProbeTime,omitempty"`
-	
-  // Last time the condition transitioned from one status to another.
+	// Last time the condition transitioned from one status to another.
 	// +optional
 	LastTransitionTime metav1.Time `json:"lastTransitionTime,omitempty"`
-	
-  // Reason is a brief CamelCase string that describes any failure
+	// Reason is a brief CamelCase string that describes any failure
 	Reason string `json:"reason"`
-	
-  // Human-readable message indicating details about last transition.
+	// Human-readable message indicating details about last transition.
 	Message string `json:"message"`
 }
 
-// CStorVolumeConfigConditionType is a valid value of CstorVolumeConfigCondition.Type
+// CStorVolumeConfigConditionType is a valid value of CstorVolumeClaimCondition.Type
```

Comment:
it is supposed to be CStorVolumeConfigCondition not CStorVolumeClaimCondition

Reply:
fixed